typing/cmt_format.ml

```diff
@@ -162,6 +162,177 @@ let record_value_dependency vd1 vd2 =
   if vd1.Types.val_loc <> vd2.Types.val_loc then
     value_deps := (vd1, vd2) :: !value_deps
 
+(* Generate an .annot file from a .cmt file. *)
```

Comment:
Cannot one instead move the existing file cmt2annot.ml into typing/ or driver/?  This would keep "legacy" code in its own module.

Alternatively, if one needs to put the code right here in cmt_format, what about exporting the feature so that it can be used directly by cmt2annot/read_cmt?

Reply:
Yes, it is possible, but it was easier to do it this way at first, dependencies-wise. I will keep the code in its original module and move it into `typing/`.